Gopkg.lock

```diff
@@ -354,6 +360,12 @@
   packages = ["xdr2"]
   revision = "1a41d1a06c93bf8a0e0385be3847a277bb793187"
 
+[[projects]]
+  branch = "master"
```

Comment:
we should not manually edit `Gopkg.lock` file, need to add this dependency if required in `Gopkg.toml` file and regenerate this file.

Reply:
when I regenerate this file, it default revision which it took was not working. so I modified the revision with the current one.

Reply:
but we should not edit lock file, anyhow you are implementing the function for displaying prompt we don't require this one.